client.go

```diff
@@ -1496,3 +1496,7 @@ func createClient(hc *http.Client) *Client {
 
 	return c
 }
+
+func CookieJar(c *Client) *http.CookieJar {
```

Comment:
@muzimu This can be accessed via -
```go
restyClient := resty.New()
...
cookieJar := restyClient.GetClient().Jar
```
I believe this PR is not required. Thanks for your time.